templates/cassiopeia/scss/blocks/_header.scss

```
@@ -66,14 +59,13 @@
 
   .mod-menu {
     flex: 1 0 100%;
-    margin: 0;
     padding: 0;
+    margin: 0;
     list-style: none;
-    color: $white;
 
     @include media-breakpoint-up(md) {
-      flex: 1 1 0%;
       display: flex;
+      flex: 1 1 0%;
```

The third value is width so when it is zero you dont use the unit or %

```suggestion
      flex: 1 1 0;
```

---

Corrected in last commit